modules/build/src/main/scala/scala/build/options/BuildOptions.scala

```diff
@@ -38,8 +38,7 @@ final case class BuildOptions(
   internal: InternalOptions = InternalOptions(),
   mainClass: Option[String] = None,
   testOptions: TestOptions = TestOptions(),
-  packageOptions: PackageOptions = PackageOptions(),
-  replOptions: ReplOptions = ReplOptions()
+  notForBloopOptions: PostBuildOptions = PostBuildOptions()
```

Comment:
I think we should call it `postBuildOptions`

Reply:
Or maybe rename `PostBuildOptions` to `NotForBloopOptions `? I wanted to make it clear that it shouldn't be used when generating the Bloop config for example.

Reply:
That's what I wanted to do initially, something like
```scala
final case class AllBuildOptions(
  buildOptions: BuildOptions = BuildOptions(),
  postBuildOption: PostBuildOptions = PostBuildOptions()
)
```

I just fear it'll touch many-many files in the codebase.